verify/test.sh

```diff
@@ -21,11 +21,20 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 cd "$KUBE_ROOT"
 
+# WARNING: we exclude the following paths from linting
+# because the CI lacks the required golang version (1.20)
+EXCLUDE_PATHS=("util-images/watch-list")
```

Comment:
should I create an issue in this repo so that we don't forget to fix it once we switch to go1.20?

Reply:
Yes, please do :)